pkg/asset/ignition/machine/node.go

```diff
@@ -35,17 +35,13 @@ func pointerIgnitionConfig(installConfig *types.InstallConfig, rootCA []byte, ro
 		// way to configure DNS before Ignition runs.
 		ignitionHost = net.JoinHostPort(installConfig.BareMetal.APIVIPs[0], "22623")
 	case nutanixtypes.Name:
-		if len(installConfig.Nutanix.APIVIPs) > 0 {
```

Comment:
This conditional seems correct, we should probably add the same check to other on-prem platforms, in case we allow for empty APIVIPs.